src/frontend/components/RangeSlider/RangeSlider.js

```diff
@@ -47,16 +51,16 @@ const RangeSlider = ({ min, max, gap }) => {
           min={min}
           max={max}
           step={100}
-          value={maxVal}
+          value={maxPrice}
           onChange={(e) => {
             e.target.value <= max &&
-              e.target.value - minVal >= gap &&
-              setMaxVal(e.target.value);
+              e.target.value - minPrice >= gap &&
+              dispatchFilters({ type: MAX_PRICE, payload: e.target.value });
           }}
         />
       </div>
       <div className="slider-labels">
-        {[...Array(30).keys()].map((value) => (
+        {[...Array(35).keys()].map((value) => (
           <span key={value}>|</span>
         ))}
```

Comment:
Here, You can use `<datalist>` and  `<option>` to show these labels and values for better semantics.

Reply:
Doing it now! Thank you.